float64 EMPTY_VELOCITY = -100.0
string EMPTY_FOLLOW_METHOD = ""
string FOLLOW_LEFT = "left"
string FOLLOW_RIGHT = "right"
string FOLLOW_CENTER = "center"

float64 velocity
string follow_method
